### DISCUSSION TITLE
Followed remote hosting tuto with HuggingFace and end up with blank page

### DISCUSSION THREAD

**Discussion by eudemocracystiftung** [danny-avila/LibreChat#4333] [Help Wanted]
Hi,

First off, thanks a lot for all your work. I am *very* new to all this, but excited to try.

Basically, our end goal is to customise a chatbot that will use the data of our website to answer specific questions (we are in the right place to start, right?).

We followed the [LibreChat tutorial](https://www.librechat.ai/docs/remote/huggingface) for remote deployment with HuggingFace and it all went smoothly. The only thing is that, when all is done, we end up with a blank page.

<img width="1143" alt="Screenshot 2024-10-04 alle 10 07 27" src="https://github.com/user-attachments/assets/e6c9fe12-81e0-417f-acfb-0e8034ff6767">

The build logs seem normal:

```
===== Build Queued at 2024-10-03 20:51:22 / Commit SHA: 088c337 =====

--> FROM ghcr.io/danny-avila/librechat-dev:latest@sha256:d27758daed5574d4ec5cfddc8bda405f81d9be76982ec58d02dd4e103be50b57
DONE 0.0s

--> RUN mkdir -p /app/api/logs/
CACHED

--> RUN mkdir -p /app/data
CACHED

--> RUN chmod -R 777 /app/uploads/temp
CACHED

--> RUN chmod -R 777 /app/data
CACHED

--> RUN curl -o /app/librechat.yaml https://raw.githubusercontent.com/fuegovic/lc-config-yaml/main/librechat-rw.yaml
CACHED

--> RUN chmod -R 777 /app/client/public/images
CACHED

--> RUN mkdir -p /app/client/public/images/temp
CACHED

--> RUN chmod -R 777 /app/api/logs/
CACHED

--> RUN mkdir -p /app/uploads/temp
CACHED

--> RUN cd /app/api && npm install
CACHED

--> Pushing image
DONE 1.0s

--> Exporting cache
DONE 0.6s
```

And the container logs too (I am only including the beginning and end for brevity):

```
===== Application Startup at 2024-10-03 20:51:29 =====


> LibreChat@v0.7.5-rc2 backend
> cross-env NODE_ENV=production node api/server/index.js

2024-10-03 20:52:11 info: [Optional] Redis not initialized. Note: Redis support is experimental.
2024-10-03 20:52:14 info: Connected to MongoDB
2024-10-03 20:52:16 info: [deleteNullOrEmptyConversations] Deleted 0 conversations and 0 messages
2024-10-03 20:52:16 info: Custom config file loaded:
2024-10-03 20:52:16 info: {
  "version": "1.1.6",
  "cache": true,
  "registration": {
    "socialLogins": [
      "discord",
      "facebook",
      "github",
      "google",
      "openid"
    ]
  },
```

and 

```
2024-10-03 20:52:16 warn: RAG API is either not running or not reachable at undefined, you may experience errors with file uploads.
2024-10-03 20:52:16 info: Updating 'USER' role MULTI_CONVO 'USE' permission from false to: true
2024-10-03 20:52:16 info: Updated 'USER' role permissions
2024-10-03 20:52:16 info: No changes needed for 'ADMIN' role permissions
2024-10-03 20:52:16 info: 
Outdated Config version: 1.1.6
Latest version: 1.1.7

      Check out the Config changelogs for the latest options and features added.

      https://www.librechat.ai/changelog


2024-10-03 20:52:17 info: Server listening on all interfaces at port 3080. Use http://localhost:3080 to access it
```

It really feels like we are missing something obvious, but, since the tutorial ends with "3. When ready, Building will change to Running. And you will be able to access LibreChat!" with an image of a login form while we do not see anything, it is not easy figuring out what is missing.

Thanks!

EDIT: any ideas about this issue?
EDIT2: anyone?

**Comment by eudemocracystiftung** [comment]
Anyone?

**Reply by fuegovic** [reply]
Did you clone the **template** as instructed in the docs or did you clone the online **demo**?

**Reply by eudemocracystiftung** [reply]
I am pretty sure I cloned the template: I used the [link provided](https://huggingface.co/spaces/LibreChat/template), then, on the right, clicked on the three-dot menu, and then on "duplicate this space". This seems like the template and not the demo, right?

At any rate, since it had been a while and I wanted to be sure, I did it all over again and get the same result. [Here](https://extrait.facil.services/?46ed24bd11e23389#17XP6yu76dze8UbgdRULPMeRgmNDkwoch9n3Wf5Vf3w) is the entire log that I get.

Is that any helpful?

**Reply by eudemocracystiftung** [reply]
Any ideas?
